admin/controllers/pupil.js

```diff
@@ -196,7 +196,9 @@ const getEditPupilById = async (req, res, next) => {
     const pupil = await pupilDataService.sqlFindOneBySlugWithAgeReason(req.params.id, req.user.schoolId)
     pupilExampleYear = pupilPresenter.getPupilExampleYear()
     if (!pupil) {
-      return next(new Error(`Pupil ${req.params.id} not found`))
+      let error = new Error(`Pupil ${req.params.id} not found`)
```

Comment:
nice to see how it would be used.  please add this to the admin/readme.md as an example, but remove from the pupil controller, as its not a requirement.